src/Http/Controllers/WysiwygMediaUploadController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Brackets\AdminUI\Http\Controllers;
+
+use Brackets\AdminUI\WysiwygMedia;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\Config;
+use Intervention\Image\Facades\Image;
+use Spatie\ImageOptimizer\OptimizerChainFactory;
+
+class WysiwygMediaUploadController extends BaseController {
+
+    public function upload(Request $request)
+    {
+        // get image from request
+        $temporaryFile = $request->file('fileToUpload');
```

Comment:
It will be better if the file will be validated. For existence, or for mimetype.